README.md

```diff
@@ -177,16 +203,25 @@ mozillavpn linuxdaemon needs privileged access and so if you do not run as
 root, you will get an authentication prompt every time you try to reconnect the
 vpn.
 
-### How to build from source code on MacOS
+### How to build from source code on macOS
+
+There are two ways to build the project on macOS - Through the terminal or with Xcode.
+For both, you must install a version of the VPN application in the main macOS Applications 
+folder so that the daemon is available for the debug build. Unless major changes were recently made, 
+you can likely use either the [latest release version](https://www.mozilla.org/en-US/products/vpn/download/) 
+or the [latest `main` branch build](https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/mozillavpn.v2.mozilla-vpn-client.branch.main.latest.build.macos/artifacts/public%2Fbuild%2FMozillaVPN.pkg).
```

Comment:
Maybe we can link the latest official build instead: https://vpn.mozilla.org/r/vpn/download/mac

Reply:
That is essentially what the `latest release version` link on the above line represents - I'm going to drop this entire line, unless you think there is a better resolution.

Reply:
Updated this